### ISSUE TITLE
KilledWorker error in the deployment

### ISSUE THREAD

**Issue by paigem** [pangeo-forge/cesm-atm-025deg-feedstock#5]
It looks like the most recent [recipe run failed again](https://pangeo-forge.org/dashboard/recipe-run/347), this time with a `KilledWorker` error. This appears to be a different failure than we saw in [this previous recipe run](https://pangeo-forge.org/dashboard/recipe-run/340), which did not result in an explicit error, but only processed a single chunk in time.

@cisaacstern Any thoughts on how to get passed a `KilledWorker` error?

**Comment by cisaacstern** [comment]
@paigem, is it possible that any of the variables added in #4 are fields which, even when divided into 5 subsets

https://github.com/pangeo-forge/cesm-atm-025deg-feedstock/blob/d19040ba9e4eedf03f06c58525bbfbe03c10d693/feedstock/recipe.py#L48

are still > 800 MB in size?

For example, if any of the variables added were >= 5 GB in size, then with our current subset inputs, each subset will be >= 1 GB in size, which could easily kill a worker.

If this is the case, the way to solve it would be to increase `subset_inputs` to an integer divisor which is large enough to make each subset <= about 800 MB.

Also, just noting that dealing with these troublesome infrastructure concerns is one of many problems which the work scoped in https://github.com/pangeo-forge/pangeo-forge-recipes/issues/256 should solve once complete.

**Comment by paigem** [comment]
Thanks @cisaacstern for explaining this.

Each individual netCDF file (1 year of daily output) is just under 1 GB. Since I'm stringing together 9 years of data, then the full time series for each variable would indeed be roughly 9 GB, which is > 800MB per chunk.

I decided to go for the `subset_inputs=5` with time chunks of length `73`, since 5 is one of the few divisors of 365. If I increase the subset_inputs to, say 12 (for roughly one-month segments), this would yield time chunks of unequal size. Would that still work?

**Comment by cisaacstern** [comment]
> this would yield time chunks of unequal size. Would that still work?

Yes, `pangeo-forge-recipes` should understand how to put the remainder into the last chunk. Let's try that and see what happens. Could you make a PR proposing this change? Thanks for your patience as we work this out, Paige.
